### ISSUE TITLE
Could not pass event EntityDamageByEntityEvent to mcMMO v2.1.135

### ISSUE THREAD

**Issue by CirnoTH** [mcMMO-Dev/mcMMO#4237]
mcMMO version: 2.1.135
Server version: Tuinity 1.16.1 (Latest build)
https://paste.ubuntu.com/p/zDgQn8wQjD/

**Comment by andris155** [comment]
Same problem.

**Comment by nossr50** [comment]
Does it happen with paper instead of tuinity?

**Comment by andris155** [comment]
```
[14:42:42] [Server thread/ERROR]: Could not pass event EntityDamageByEntityEvent to mcMMO v2.1.136
java.lang.NullPointerException: null
	at com.gmail.nossr50.util.ItemUtils.isArmor(ItemUtils.java:103) ~[?:?]
	at com.gmail.nossr50.skills.axes.Axes.hasArmor(Axes.java:30) ~[?:?]
	at com.gmail.nossr50.skills.axes.AxesManager.canImpact(AxesManager.java:46) ~[?:?]
	at com.gmail.nossr50.util.skills.CombatUtils.processAxeCombat(CombatUtils.java:144) ~[?:?]
	at com.gmail.nossr50.util.skills.CombatUtils.processCombatAttack(CombatUtils.java:378) ~[?:?]
	at com.gmail.nossr50.listeners.EntityListener.onEntityDamageByEntity(EntityListener.java:428) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor715.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.1.jar:git-Paper-127]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.16.1.jar:git-Paper-127]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.1.jar:git-Paper-127]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.1.jar:git-Paper-127]
	at org.bukkit.craftbukkit.v1_16_R1.event.CraftEventFactory.callEvent(CraftEventFactory.java:247) ~[patched_1.16.1.jar:git-Paper-127]
	at org.bukkit.craftbukkit.v1_16_R1.event.CraftEventFactory.callEntityDamageEvent(CraftEventFactory.java:1035) ~[patched_1.16.1.jar:git-Paper-127]
	at org.bukkit.craftbukkit.v1_16_R1.event.CraftEventFactory.handleEntityDamageEvent(CraftEventFactory.java:922) ~[patched_1.16.1.jar:git-Paper-127]
	at org.bukkit.craftbukkit.v1_16_R1.event.CraftEventFactory.handleEntityDamageEvent(CraftEventFactory.java:878) ~[patched_1.16.1.jar:git-Paper-127]
	at org.bukkit.craftbukkit.v1_16_R1.event.CraftEventFactory.handleLivingEntityDamageEvent(CraftEventFactory.java:1067) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.EntityLiving.damageEntity0(EntityLiving.java:1823) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.EntityHuman.damageEntity0(EntityHuman.java:844) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.EntityLiving.damageEntity(EntityLiving.java:1159) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.EntityHuman.damageEntity(EntityHuman.java:761) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.EntityPlayer.damageEntity(EntityPlayer.java:833) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.EntityHuman.attack(EntityHuman.java:1092) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.EntityPlayer.attack(EntityPlayer.java:1861) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.PlayerConnection.a(PlayerConnection.java:2139) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.PacketPlayInUseEntity.a(PacketPlayInUseEntity.java:49) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.PacketPlayInUseEntity.a(PacketPlayInUseEntity.java:6) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:23) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.TickTask.run(SourceFile:18) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.MinecraftServer.aZ(MinecraftServer.java:1137) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.MinecraftServer.executeNext(MinecraftServer.java:1130) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.MinecraftServer.sleepForTick(MinecraftServer.java:1091) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:1005) ~[patched_1.16.1.jar:git-Paper-127]
	at net.minecraft.server.v1_16_R1.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.1.jar:git-Paper-127]
	at java.lang.Thread.run(Unknown Source) [?:?]
```

I use Paper.

**Comment by nossr50** [comment]
Hmm, last time I looked over this issue I wasn't able to find any cause, I'll look again.

**Comment by nossr50** [comment]
Should be fixed in commit 9f9518eea04daa9355edc8c8c998f6f355bc54ca

**Comment by andris155** [comment]
Thanks
